include/inexor/vulkan-renderer/world/indentation.hpp

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <cstdint>
+
+namespace inexor::vulkan_renderer::world {
+
+class Indentation {
+public:
+    static constexpr std::uint8_t MAX = 8;
+
+private:
+    std::uint8_t m_start = 0;
+    std::uint8_t m_end = Indentation::MAX;
+
+public:
+    Indentation() = default;
+    Indentation(const std::uint8_t start, const std::uint8_t end) noexcept;
+    explicit Indentation(const std::uint8_t uid) noexcept;
```

Comment:
```suggestion
    Indentation(std::uint8_t start, std::uint8_t end) noexcept;
    explicit Indentation(std::uint8_t uid) noexcept;
```

Reply:
Fixed in d81c9e9576c42a3b00b817bb4727c44dc02ff2d6